appconfiguration/azure-spring-cloud-feature-management/dynamic-features-sample/src/main/resources/application.yml

```diff
@@ -0,0 +1,34 @@
+feature-management:
+  dynamic-features:
+    ShoppingCart:
+      assigner: Microsoft.Targeting
+      variants:
+      - 
+        default: true
+        name: Big
+        configuration-reference: ShoppingCart.Big
+        assignmentParameters:
+          Audience:
+            users:
+            - 
+              Alec
+            groups: []
+      - 
+        name: Small
+        configurationReference: ShoppingCart.Small
+        assignment-parameters:
+          Audience:
+            users: []
+            groups:
+            - 
+              name: Ring1
+              rolloutPercentage: 50
+            defaultRolloutPercentage: 30
```

Comment:
Same to all other places. Use Kebab case.
Refs: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#features.external-config.typesafe-configuration-properties.relaxed-binding